config/config.local.sample.neon

```diff
@@ -38,6 +39,7 @@ parameters:
 
 	s3:
 		enabled: true # disable on dev, enable on production
+		basePath: # shared-beta/domain.mgw.cz
```

Comment:
- Chtělo by zařídit, aby tohle vyplnoval asi #server, jinak se na to bude imho zapomínat.
- výše se nastavuje stage v `parameters`, což by asi mělo být to samé co tady v `shared-{stage}`?

Reply:
Pokud vím, tak config nikdy nepřipravuje #server nebo se pletu?

Stage u `basePath` nemusí být stejná. Například chci mít stage při vývoji `local`, ale používat média z bety, proto si nastavím `basePath` na `shared-beta` 

Na k8s už je basePath stejně vždy nastavená a bez ni to nebude fungovat, takže se nezapomene.

Ano, správné řešení je mít pre-deploy check, který zkontroluje config.local oproti config.sample a tím se udrží konzistence, ale ještě to nikdo nepřipravil...